include/uacpi/platform/compiler.h

```diff
@@ -25,7 +25,7 @@
     #define UACPI_PACKED(decl) decl __attribute__((packed));
 #endif
 
-#ifdef __GNUC__
+#if defined(__GNUC__) || defined(__clang__)
```

Comment:
I think clang defines `__GNUC__` as well, which is why this check wasn't there in the first place. Are you sure it's needed here?

Reply:
Clang in MSVC mode doesn't define `__GNUC__`.

Reply:
Interesting